### PR TITLE
Add a silent flag to remove deployment log output

### PR DESCRIPTION
**Pull request by Lbqds**
By default, the `CLI` will output the deployment log. Users can use the `--silent` flag to remove the deployment log output, which is similar to Hardhat.

### CODE REVIEW

File: packages/cli/src/deployment.ts [hunk 1]
```diff
@@ -369,16 +387,16 @@ function createDeployer<Settings = unknown>(
...
       const contractInstance = contractFactory.at(previousDeployResult.contractInstance.address)
       return {
         ...previousDeployResult,
         contractInstance
       }
     }
-    console.log(`Deploying contract ${taskId}`)
-    console.log(`Deployer - group ${signer.group} - ${signer.address}`)
+    deploymentLogger(`Deploying contract ${taskId}`)
+    debugLogger(`Deployer - group ${signer.group} - ${signer.address}`)
```
Comment (Lbqds) [inline packages/cli/src/deployment.ts:399]: We only log the deployer info in debug mode.

**Review by h0ngcha0 — APPROVED** [review]
LGTM :+1:

File: packages/cli/src/deployment.ts [hunk 2]
```diff
@@ -323,7 +323,14 @@ function getTaskId(code: Contract | Script, taskTag?: string): string {
...
 }
 
+function debugLog(enableDebugMode: boolean, message: string) {
+  if (enableDebugMode) {
+    console.log(message)
+  }
+}
+
 function createDeployer<Settings = unknown>(
+  enableDebugMode: boolean,
```
Comment (polarker) [inline packages/cli/src/deployment.ts]: `debugModeEnabled: boolean`

**Review by polarker — APPROVED** [review]
LGTM

**Pull request merged**